packages/main/src/themes/base/CheckBox.less

```diff
@@ -9,304 +9,216 @@
 /* CSS for control sap.m/CheckBox  */
 /* Base theme                      */
 /* =============================== */
-@_ui5_checkbox_height: 3rem;
-@_ui5_checkbox_compact_height: 2rem;
-@_ui5_checkbox_square_hover_bg: @sapUiFieldHoverBackground;
-@_ui5_checkbox_square_hover_border_color: @sapUiFieldHoverBorderColor;
-@_ui5_checkbox_square_outline_width: 1px;
-@_ui5_checkbox_square_border_width: 0.125rem;
-@_ui5_checkbox_square_error_border_width: 0.125rem;
-@_ui5_checkbox_square_warning_border_width: 0.125rem;
-@_ui5_checkbox_square_error_border_style: solid;
-@_ui5_checkbox_square_warning_border_style: solid;
-@_ui5_checkbox_square_disabled_border_color: @sapUiFieldBorderColor;
-@_ui5_checkbox_square_disabled_bg_color: @sapUiFieldBackground;
-@_ui5_checkbox_square_disabled_error_border_color: @sapUiFieldInvalidColor;
-@_ui5_checkbox_square_border_radius: 0;
-@_ui5_checkbox_checkmark_color: @sapUiSelected;
+
+@_ui5_checkbox_width_height: 3rem;
+@_ui5_checkbox_compact_width_height: 2rem;
+@_ui5_checkbox_hover_background: @sapUiFieldHoverBackground;
+@_ui5_checkbox_inner_width_height: 1.375rem;
+@_ui5_checkbox_inner_border: solid .125rem @sapUiFieldBorderColor;
+@_ui5_checkbox_inner_border_radius: 0;
+@_ui5_checkbox_inner_error_border: 0.125rem solid @sapUiFieldInvalidColor;
+@_ui5_checkbox_inner_warning_border: 0.125rem solid @sapUiFieldWarningColor;
+@_ui5_checkbox_inner_readonly_border: 0.125rem solid @sapUiFieldReadOnlyBorderColor;
 @_ui5_checkbox_checkmark_warning_color: darken(@sapUiFieldWarningColor, 100);
-@_ui5_checkbox_checkmark_line_height: inherit;
-@_ui5_checkbox_checkmark_compact_line_height: inherit;
-@_ui5_checkbox_checkmark_compact_height: 0.75rem;
-@_ui5_checkbox_checkmark_compact_width: 0.75rem;
-@_ui5_checkbox_focus_border_width: 1px;
-@_ui5_checkbox_focus_outline_element_top: 0.6875rem;
-@_ui5_checkbox_focus_outline_element_bottom: 0.6875rem;
+@_ui5_checkbox_checkmark_color: @sapUiSelected;
+@_ui5_checkbox_focus_left_top_bottom_position: .6875rem;
+@_ui5_checkbox_compact_no_label_focus_right: 0px;
+@_ui5_checkbox_no_label_focus_right: .6875rem;
+@_ui5_checkbox_focus_outline: 1px dotted @sapUiContentFocusColor;
+@_ui5_checkbox_compact_inner_size: 1rem;
+@_ui5_checkbox_compact_focus_left_top_bottom_position: .375rem;
+@_ui5_checkbox_wrapped_content_margin_top: 0.125rem;
+@_ui5_checkbox_wrapped_focus_padding: .375rem;
 
 :host(ui5-checkbox) {
 	display: inline-block;
 	overflow: hidden;
 	max-width: 100%;
 }
 
-// required for browsers with no shadow DOM native support
 ui5-checkbox {
 	display: inline-block;
 	overflow: hidden;
 	max-width: 100%;
 }
 
-.sapMCb {
+.ui5-checkbox-wrapper {
+	min-height: @_ui5_checkbox_width_height;
```

Comment:
add the css property  -webkit-tap-highlight-color: rgba(0,0,0,0)
to remove highlighting upon click on iOS